routing/unified_policies.go

```diff
@@ -223,6 +224,18 @@ func (u *unifiedPolicy) getPolicyNetwork(
 			continue
 		}
 
+		// Track the maximal capacity for usable channels. If we don't
+		// know the capacity, we fall back to MaxHTLC.
+		capMsat := lnwire.NewMSatFromSatoshis(edge.capacity)
+
+		if capMsat == 0 && edge.policy.MessageFlags.HasMaxHtlc() {
+			capMsat = edge.policy.MaxHTLC
+		}
+
+		if capMsat > maxCapMsat {
```

Comment:
Non blocking, but could be a chance to start to introduce some more type params here now that our min version is Go 1.8:
```go
func Max[N constraints.Integer](num1 N, num2 N) N {
     if num1 > num2 {
         return num1
     }
     
     return num2
}
```

In terms of where it should live, we can prob just throw it in `lntypes` for now. In the taro, repo we've started to accumulate some other nice generic channel send/recv timeout funcs: https://github.com/lightninglabs/taro/tree/main/chanutils